### PR TITLE
Fix number format

### PR DESCRIPTION
**Pull request by kdeininger**
Refs #405

### CODE REVIEW

File: print/print-apps/oereb/topiclegend.jrxml [hunk 1]
```diff
@@ -44,7 +44,7 @@
 				<textElement textAlignment="Right" verticalAlignment="Top" markup="html">
 					<font fontName="Cadastra" size="9"/>
 				</textElement>
-				<textFieldExpression><![CDATA[java.text.MessageFormat.format("{0} m<sup>2</sup>", java.text.NumberFormat.getInstance($P{REPORT_LOCALE}).format($F{Area}))]]></textFieldExpression>
+				<textFieldExpression><![CDATA[java.text.MessageFormat.format("{0} m<sup>2</sup>", String.valueOf($F{Area}))]]></textFieldExpression>
```
Comment (sbrunner) [inline print/print-apps/oereb/topiclegend.jrxml]: Why this change?

File: print/print-apps/oereb/mainpage.jrxml [hunk 2]
```diff
@@ -219,7 +219,7 @@
 				<textElement verticalAlignment="Middle" markup="html">
 					<font fontName="Cadastra" size="9"/>
 				</textElement>
-				<textFieldExpression><![CDATA[$P{RealEstate_LandRegistryArea} + " m<sup>2</sup>"]]></textFieldExpression>
+				<textFieldExpression><![CDATA[Integer.toString(Math.round(Float.parseFloat($P{RealEstate_LandRegistryArea}))) + " m<sup>2</sup>"]]></textFieldExpression>
```
Comment (sbrunner) [inline print/print-apps/oereb/mainpage.jrxml]: Why not:
```java
NumberFormat.getIntegerInstance().format(Float.parseFloat($P{RealEstate_LandRegistryArea}))
```

Comment (sbrunner) [reply print/print-apps/oereb/mainpage.jrxml]: Even better:
```
java.text.MessageFormat.format("{0,number,0} m<sup>2</sup>", $P{RealEstate_LandRegistryArea})
```
and RealEstate_LandRegistryArea should be a number ...

File: print/print-apps/oereb/topiclegend.jrxml (hunk 1 above)
Comment (sbrunner) [inline print/print-apps/oereb/topiclegend.jrxml]: =>
```
java.text.MessageFormat.format("{0,number,0} m<sup>2</sup>", $F{Area})
```

File: print/print-apps/oereb/mainpage.jrxml (hunk 2 above)
Comment (vvmruder) [reply print/print-apps/oereb/mainpage.jrxml]: 👍

File: print/print-apps/oereb/topiclegend.jrxml (hunk 1 above)
Comment (vvmruder) [reply print/print-apps/oereb/topiclegend.jrxml]: 👍

**Comment by vvmruder** [comment]
Please have a look at #458

**Comment by kdeininger** [comment]
Fixed with #466
